lib/convertkit/client.rb

```diff
@@ -4,6 +4,8 @@ module ConvertKit
   # The ConvertKit OAuth class can be used to obtain this token.
   class Client
     API_URL = 'https://api.convertkit.com/alpha'.freeze # Using Alpha API as this is currently in active development
+    HTTP_METHODS = %i[get post delete].freeze
```

Comment:
You may as well add `put` here since some of their API calls use PUT

```suggestion
    HTTP_METHODS = %i[get post delete put].freeze
```

![CleanShot 2023-09-01 at 10 09 08](https://github.com/untitledstartup/convertkit-api-ruby/assets/5408839/a9962731-997f-4951-9b9e-ad9b5c4f22a9)

Good use of metaprogramming!